README.md

```diff
@@ -16,11 +16,7 @@ A few things you might want to know about `jiant`:
 
 To run a simple example demo experiment using data from GLUE:
 
-1. Clone `jiant` and its submodules.
-2. Run the dependencies file.
-3. Download the GLUE data.
-4. Set up environment variables to point to your directories for data and experiment output.
-5. Run demo.sh.
+Follow this [getting started tutorial]()! 
```

Comment:
Fix.

Reply:
I'll put in the official link once the tutorials are checked in